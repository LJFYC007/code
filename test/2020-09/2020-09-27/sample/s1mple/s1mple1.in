5
00100
00010
10001
01000
00100
4
0011
1010
0000
1001
