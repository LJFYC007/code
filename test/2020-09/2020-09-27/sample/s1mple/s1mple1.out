0
12
28
8
